ethergo/submitter/config/config.go

```diff
@@ -67,8 +67,8 @@ const (
 // DefaultMaxPrice is the default max price of a tx.
 var DefaultMaxPrice = big.NewInt(500 * params.GWei)
 
-// DefaultBaseGasPrice is the default max price of a tx.
-var DefaultBaseGasPrice = big.NewInt(1 * params.GWei)
+// DefaultMinGasPrice is the default min price of a tx.
+var DefaultMinGasPrice = big.NewInt(10 * params.GWei)
```

Comment:
Min gas price of 10 Gwei is an overshoot.

Reply:
This makes sense- I'm confused of the intention here though in the base PR?

https://github.com/synapsecns/sanguine/pull/2432/files#diff-9cb5c46a90d8a5d8f06ab1c53594fbc934b93eaf5131c538424f0c4666f94f49R429

To me this seems like we will never set gas price below 10 gwei if no `prevTx` is supplied

Reply:
The unit there is in wei

Reply:
Let's introduce a min value for GasPrice/GasFeeCap (DefaultMinGasPrice) and a separate one for GasTipCap. The latter one is set to 10 Wei (aka just 10) in the base PR.